sympy/ntheory/partitions_.py

```diff
@@ -1,40 +1,102 @@
 from __future__ import print_function, division
-
 from mpmath.libmp import (fzero,
     from_man_exp, from_int, from_rational,
     fone, fhalf, bitcount, to_int, to_str, mpf_mul, mpf_div, mpf_sub,
-    mpf_add, mpf_sqrt, mpf_pi, mpf_cosh_sinh, pi_fixed, mpf_cos)
+    mpf_add, mpf_sqrt, mpf_pi, mpf_cosh_sinh, pi_fixed, mpf_cos, mpf_sin)
 from sympy.core.numbers import igcd
 import math
 from sympy.core.compatibility import range
+from .residue_ntheory import (_sqrt_mod_prime_power,
+    legendre_symbol, jacobi_symbol, is_quad_residue)
+
+_factor = [] # For storing the smallest prime factor
+_totient = []
+
+def pre():
+    maxn = 10**5
```

Comment:
This could be an argument (with default value). It would then be possible to change the size (by a keyword argument in ``npartitions``, for example, though not necessarily in this PR).